connectors/src/connectors/google_drive/temporal/activities.ts

```diff
@@ -389,14 +390,20 @@ async function syncOneFile(
   });
 
   if (documentContent.length <= MAX_DOCUMENT_TXT_LEN) {
+    const parents = await getFileParentsMemoized(
```

Comment:
Does it return a parent list as per the specs of the parents field [here](https://github.com/dust-tt/dust/blob/main/core/src/data_sources/data_source.rs) line 93